rest_api/actix_web_4/Cargo.toml

```diff
@@ -25,7 +25,9 @@ description = """\
 [dependencies]
 actix-utils = "3"
 actix-web = "4"
-splinter = { path = "../../libsplinter", features = ["store", "store-factory"] }
+futures = "0.3"
+splinter = { path = "../../libsplinter", features = ["store","store-factory","rest-api"] }
```

Comment:
This crate and -common are meant to replace rest-api code in splinter so this dependency isn't desired.

Reply:
Right you are, Ill pull the dependency out.